app/lib/activitypub/tag_manager.rb

```diff
@@ -86,7 +93,16 @@ def cc(status)
       cc << COLLECTIONS[:public]
     end
 
-    cc.concat(status.active_mentions.map { |mention| uri_for(mention.account) }) unless status.direct_visibility? || status.limited_visibility?
+    unless status.direct_visibility? || status.limited_visibility?
+      if status.account.silenced?
+        # Only notify followers if the account is locally silenced
+        account_ids = status.active_mentions.pluck(:account_id)
+        cc.concat(status.account.followers.where(id: account_ids).map { |account| uri_for(account) })
+        cc.concat(FollowRequest.where(target_account_id: status.account_id, account_id: account_ids).map { |request| uri_for(request.account) })
```

Comment:
what's up with this line?

Reply:
I'm not sure what you mean. It's to allow silenced local users to mention (and notify) remote users who have requested to follow them.
This is the same logic as in `NotifyService` for local notifications.